File: src/main/java/intro/_02_java_interop/TextColorFactory.java
package intro._02_java_interop;

import java.awt.*;

public class TextColorFactory {

    public static Color fromRgb(int red, int green, int blue) {
        return fromRgb(red, green, blue, 255);
    }

    public static Color fromRgb(int red, int green, int blue, int alpha) {
        return new Color(red, green, blue, alpha);
    }
}

File: src/main/kotlin/hu/appcraft/kotlin/workshop/intro/_02_java_interop/TextColorFactory.kt
package hu.appcraft.kotlin.workshop.intro._02_java_interop

import java.awt.Color

/**
 * You can check this class in its original version at:
 * https://github.com/Hexworks/zircon/blob/master/src/main/kotlin/org/codetome/zircon/api/color/TextColorFactory.kt
 */
object TextColorFactory {

    // TODO: modify this method to use `@JvmOverloads`
    // TODO: and add a default value (255) for `alpha`
    fun fromRGB(red: Int, green: Int, blue: Int, alpha: Int): Color
            = Color(red, green, blue, alpha)

}